sql/1_0_1-to-2_0_0_upgrade.sql

```diff
@@ -386,3 +386,573 @@ ALTER TABLE `background_services` CHANGE `running` `running` TINYINT(1) NOT NULL
+('Issues|/interface/patient_file/problem_encounter.php', 'Issues', '', '', '', 'tre', '/interface/patient_file/problem_encounter.php', 1, NULL, NULL),
+('Lab Documents|/interface/main/display_documents.php', 'Lab Documents', '', '', '', 'pat', '/interface/main/display_documents.php', 0, 'null', 'null'),
+('Lab Overview|/interface/patient_file/summary/labdata.php', 'Lab Overview', '', '', '', 'enc', '/interface/patient_file/summary/labdata.php', 1, 'null', 'null'),
+('Language|/interface/language/language.php', 'Language', '', '', '', 'adm', '/interface/language/language.php', 0, '["admin","language"]', 'null'),
+('Layouts|/interface/super/edit_layout.php', 'Layouts', '', '', '', 'adm', '/interface/super/edit_layout.php', 0, '["admin","super"]', 'null'),
+('Lists|/interface/super/edit_list.php', 'Lists', '', '', '', 'adm', '/interface/super/edit_list.php', 0, '["admin","super"]', 'null'),
+('List|/interface/reports/inventory_list.php', 'List', '', '', '', 'rep', '/interface/reports/inventory_list.php', 0, 'null', 'null'),
+('List|/interface/reports/patient_list.php', 'List', '', '', '', 'rep', '/interface/reports/patient_list.php', 0, 'null', 'null'),
+('Load Compendium|/interface/orders/load_compendium.php', 'Load Compendium', '', '', '', 'pat', '/interface/orders/load_compendium.php', 0, 'null', 'null'),
+('Logs|/interface/logview/logview.php', 'Logs', '', '', '', 'adm', '/interface/logview/logview.php', 0, '["admin","users"]', 'null'),
+('Manage Modules|/interface/modules/zend_modules/public/Installer', 'Manage Modules', '', '', '', 'pat', '/interface/modules/zend_modules/public/Installer', 0, 'null', 'null'),
+('Management|/interface/drugs/drug_inventory.php', 'Management', '', '', '', 'pat', '/interface/drugs/drug_inventory.php', 0, 'null', 'null'),
+('Menu:admins', 'Menu', '', '', '', NULL, NULL, 0, '["super","admin"]', NULL),
+('Merge Patients|/interface/patient_file/merge_patients.php', 'Merge Patients', '', '', '', 'adm', '/interface/patient_file/merge_patients.php', 0, 'null', 'null'),
+('Messages|/interface/main/messages/messages.php?form_active=1', 'Messages', '', '', '', 'msg', '/interface/main/messages/messages.php?form_active=1', 0, 'null', 'null'),
+('Miscellaneous:misimg', 'Miscellaneous', '', '', '', NULL, NULL, 0, 'null', 'null'),
+('Modules:modimg', 'Modules', '', '', '', NULL, NULL, 0, 'null', 'null'),
+('Native Data Loads|/interface/super/load_codes.php', 'Native Data Loads', '', '', '', 'adm', '/interface/super/load_codes.php', 0, '["admin","super"]', 'null'),
+('New Documents|/controller.php?document&list&patient_id=00', 'New Documents', '', '', '', 'msc', '/controller.php?document&list&patient_id=00', 0, 'null', 'null'),
+('New/Patient|/interface/new/new.php', 'Add Patient', 'fa-group', '', '', 'pat', '/interface/new/new.php', 0, 'null', 'null'),
+('Ofc Notes|/interface/main/onotes/office_comments.php', 'Ofc Notes', '', '', '', 'msc', '/interface/main/onotes/office_comments.php', 0, 'null', 'null'),
```

Comment:
onotes is deprecated

Reply:
Sorry about that, I was using the sql table generation given to me, wasn't aware of the deprecated ones. I'll push a fix asap